include/cantera/oneD/Domain1D.h

```diff
@@ -348,16 +359,73 @@ class Domain1D
         return x[index(n,j)];
     }
 
+    /**
+     * Retrieve component values.
+     * @param component  name of component
+     * @param[out] values  vector of values
+     *
+     * @since New in %Cantera 3.2.
+     */
+    virtual void getValues(const string& component, vector<double>& values) const {
+        throw NotImplementedError("Domain1D::setProfile", "Needs to be overloaded.");
+    }
```

Comment:
```suggestion
    virtual void getValues(const string& component, vector<double>& values) const {
        throw NotImplementedError("Domain1D::getValues", "Needs to be overloaded.");
    }
```
A similar fix is needed for `setValues`.